content/hi/docs/contribute/localization_hi.md

```diff
@@ -0,0 +1,162 @@
+---
+title: कुबेरनेट्स प्रलेखन हिंदी स्थानीयकरण गाइड
+content_type: concept
+---
+
+<!-- overview -->
+कुबेरनेट्स प्रलेखन हिंदी स्थानीयकरण गाइड
+
+<!-- body -->
+# शैली मार्गदर्शिका
+## कुछ सामान्य दिशानिर्देश
+---
+* ### अनुवाद को सांवादिक बनाएं, शब्दशः नहीं।
+    जब हम अनुवाद करते हैं, तो हमें बस एक वाक्य के हर शब्द को हिंदी में बदलकर वापस लिखने का मतलब नहीं होता। बल्कि हमें सोचना चाहिए कि वाक्य उपयोगकर्ता को किस प्रकार का अर्थ पहुंचाने की कोशिश कर रहा है, दिए गए संदर्भ में। और उपयोगकर्ता को वाक्य पढ़ने के पश्चात क्या करने की उम्मीद है। अनुवाद का मतलब है वाक्य के सन्दर्भ, अर्थ और उद्देश्य को सही ढंग से पहुंचाना, चाहे वाक्य मूल से पूरी तरह अलग ही क्यों न हो।
+
+* ### कार्यात्मकता महत्वपूर्ण है, भाषाई गौरव नहीं।
+  हाँ, हिंदी एक शानदार भाषा है जिसका विशाल इतिहास है, और अपनी भाषाई परंपरा पर गर्व करना ठीक है।
+
+  लेकिन, एक सॉफ़्टवेयर को उद्देश्य के लिए बनाया जाता है और जो कुछ इसकी क्षमता और उपयोग में बाधा डालता है, वह व्यर्थ होगा। केवल इसलिए कि हिंदी में एक शब्द का अनुवाद मौजूद है, इसका यह मतलब नहीं है कि हमें उसे उपयोग करना चाहिए। ```कुबेरनेट्स``` को क्षमता-नियंत्रण पदार्थ में अनुवाद करना शायद हिंदी के प्रति अधिक नुकसानदायक होगा क्योंकि इससे उपयोगकर्ताओं को भ्रमित किया जाएगा और कुबेरनेट्स को हिंदी समझने से रोकेगा। आम तौर पर आप किस बात को प्रयोग करना सही समझेंगे? _"एक क्षमता-नियंत्रण पदार्थ और मूषक-यंत्र।" या "एक कीबोर्ड और माउस।"?_
+
+  जो कुछ एक शैक्षणिक या रचनात्मक संदर्भ में काम करता है, वह सॉफ़्टवेयर में काम नहीं करेगा जो वास्तविकता में सभी प्रकार के शैक्षिक, सामाजिक और सांस्कृतिक पृष्ठभूमि वाले लोगों द्वारा उपयोग किया जाने की उम्मीद की जाती है। हमेशा विचार करें कि दैनिक संवादात्मक स्थिति में क्या उपयोग होने की संभावना है, और क्या एक साधारण बातचीत की स्थिति में शामिल होना चाहिए, जबकि किताब के पाठ्यक्रम या आधिकारिक सर्कुलर/नोटिस में क्या शामिल होना चाहिए।
+
+## हिंदी अनुवाद के दौरान, सरल और सुलभ शब्दों और वाक्यांशों का उपयोग करें।
+---
+कुबेरनेट्स डॉक्स को अनावश्यक रूप से औपचारिक लहजे से बचना चाहिए। यहां कुछ उदाहरण दिए गए हैं बचने के लिए शब्द और वाक्यांश
```

Comment:
Hi @AmanSarraf , these words at same time can be use for many context, most of us native speakers will not use these perfect words every time, because then there are dialects to, maybe many of us use many translating software ( ex. google transalation) which can be use by contributours. So instead of binding hindi words by which to use or not, may be we can assess the context and then decide should we use it or not.